src/integrationTest/java/uk/gov/hmcts/dts/fact/repositories/ServiceRepositoryTest.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.dts.fact.repositories;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import uk.gov.hmcts.dts.fact.model.Service;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@DataJpaTest
+class ServiceRepositoryTest {
+
+    @Autowired
+    private ServiceRepository serviceRepository;
+
+    @Test
+    void shouldFindExistingCourt() throws IOException {
+
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
I noticed that final has not been used all the time, maybe the team should decide if it should be used and where.